### PR TITLE
New package: mupen64plus-qt-1.11

### PR DESCRIPTION
**Pull request by Vlad-Romanov**

### CODE REVIEW

**Comment by Vlad-Romanov** [comment]
I still am getting use to this, very sorry if you end up with accidental PR's.

File: srcpkgs/mupen64plus-qt/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'mupen64plus-qt'
+pkgname=mupen64plus-qt
+version=1.11
+revision=1
+build_style=qmake
+hostmakedepends="qt5-qmake qt5-host-tools"
+makedepends="quazip-devel quazip-qt5-devel qt5-tools-devel"
+depends="quazip quazip-qt5 mupen64plus hicolor-icon-theme qt5-plugin-sqlite"
+short_desc="a front-end for the mupen64plus emulator"
```
Comment (cr6git) [inline srcpkgs/mupen64plus-qt/template]: `short_desc` should start uppercase

File: srcpkgs/mupen64plus-qt/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+maintainer="VladTheImpaler <vladimir-romanov-ii@yandex.com>"
+license="BSD-1"
```
Comment (cr6git) [inline srcpkgs/mupen64plus-qt/template]: `license="BSD-3-Clause"`

**Comment by Vlad-Romanov** [comment]
Is this good now, cr6? Thank you for reviewing, am still figuring out licensing & things.

**Comment by abenson** [comment]
The commits should be squashed down to a single commit, with a message like "New package: mupen64plus-qt-1.11"

**Comment by cr6git** [comment]
@Vlad-Romanov to commit your changes you can use:
````
git commit -m "New package: mupen64plus-qt-1.11"
````
OR:
````
xbump mupen64plus-qt
````
The two commands are the same, `xbump` is part of `xtools` package.

**Comment by Vlad-Romanov** [comment]
I have made appropriate changes, is this good? I do change from web browser, must I still git commit -m ?

**Comment by cr6git** [comment]
No, not good.
As @abenson told you before, you must squash your commits to **only one.**
It's the same problem on all your PRs.

**Comment by Vlad-Romanov** [comment]
I must squash my commits to only one? So I must re-do the PR's? :/

**Comment by abenson** [comment]
[Squashing Commits](https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit)

As for the vlicense, in `do_install()` add `vlicense LICENSE`

**Comment by Vlad-Romanov** [comment]
Is this good? I merge the branch after fixing it, or is this still bad? :( 
Should I make new PR maybe? I am sorry I do not know how github works, did not think would be so difficult... feels like I have failed at this... Is this possible to do in github.com web browser? Or I MUST use git command?

**Comment by Vlad-Romanov** [comment]
Why did merge branch fail? :( ....... I will never figure it out....

**Pull request closed without merging**